R/latent_model.R

```diff
@@ -168,13 +165,45 @@ epidist_gen_log_lik_latent <- function(family) {
 #' @export
 epidist_formula_model.epidist_latent_model <- function(
     data, formula, ...) {
-  # data is only used to dispatch on
+  # Update main formula
   formula <- stats::update(
     formula, delay | vreal(relative_obs_time, pwindow, swindow) ~ .
   )
+
+  # Only update pwindow/swindow formulas if intercept only
+  fixed_dpars <- names(formula$pfix)
+  formula_dpars <- names(formula$pforms)
+
+  # Check if pwindow needs updating
```

Comment:
Is the point of this to make sure that the formula for `pwindow` and `swindow` are basically random effects on `.row_id`? Do we cover all cases so that someone can't set another formula? Or as it looks like now if someone puts something other than `pwindow ~ 1` then that will be kept? Are there use cases for something other than `pwindow ~ 0 + as.factor(.row_id)`?